packages/manager/src/features/Databases/DatabaseLanding/DatabaseRow.tsx

```diff
@@ -28,22 +31,32 @@ export const databaseEngineMap: Record<Engine, string> = {
 interface Props {
   database: Database | DatabaseInstance;
   events?: Event[];
+  isADatabases?: boolean;
```

Comment:
I'm not sure what `isADatabases` means. Can we make the name more descriptive? Maybe something like `isAivenDatabase`?

```suggestion
  isAivenDatabase?: boolean;
```

Reply:
Renamed to 'isNewDatabase'